troubleshooting/tools/tracevol.py

```diff
@@ -125,27 +151,40 @@ def format_table(arg, pvc_data, table):
         table.add_row([pvcname, pvname, "", False,
                        False, False])
         return
-    # check image details present rados omap
-    pv_present, uuid_present = validate_volume_in_rados(
-        arg, image_id, pvname, pool_name)
-    image_in_cluster = check_image_in_cluster(arg, image_id, pool_name)
+    # check image/subvolume details present rados omap
+    pv_present, uuid_present = validate_volume_in_rados(arg,
+                                                        image_id,
+                                                        pvname,
+                                                        pool_name,
+                                                        is_rbd)
+    present_in_cluster = False
+    if is_rbd:
+        present_in_cluster = check_image_in_cluster(arg, image_id, pool_name)
+    else:
+        subvolname = "csi-vol-%s" % image_id
```

Comment:
This can be configured isnt it based on the storageclass param?

Reply:
ah i see that you mention you will fix this in separate PR ,can you please open an issue

Reply:
https://github.com/ceph/ceph-csi/issues/1143